README.md

```diff
@@ -482,16 +500,37 @@ It's important to know that in order to use congestion metrics we need to fetch
 
 For both transaction types if any of the steps fails, we fallback to hardcoded values.
 
+### DOT graphs
+There are multiple ways of visualising DOT graphs:
+* `xdot` application [recommended]
+* VSCode Extensions
+* online viewers
+
+### xdot
+To install simply run `hombrew install xdot` and then run `xdot <path_to_dot_file>`. This tool seems to be the best for the job, since the viewer is interactive and supports tooltips, which in our case contain extra tracing information.
```

Comment:
typo, homebrew

Reply:
Also, consider `nix` https://search.nixos.org/packages?channel=unstable&show=xdot&from=0&size=50&sort=relevance&type=packages&query=xdot

Reply:
Not sure it makes sense as `nix develops` errors, when trying to install `xvfb`:
```
error: Package ‘xvfb-run-1+g87f6705’ in /nix/store/byybjfdwhkpv18xqj7945ca755gbisc3-source/pkgs/tools/misc/xvfb-run/default.nix:59 is not available on the requested hostPlatform:
         hostPlatform.config = "aarch64-apple-darwin"
         package.meta.platforms = [
           "aarch64-linux"
           "armv5tel-linux"
           "armv6l-linux"
           "armv7a-linux"
           "armv7l-linux"
           "i686-linux"
           "loongarch64-linux"
           "m68k-linux"
           "microblaze-linux"
           "microblazeel-linux"
           "mips-linux"
           "mips64-linux"
           "mips64el-linux"
           "mipsel-linux"
           "powerpc64-linux"
           "powerpc64le-linux"
           "riscv32-linux"
           "riscv64-linux"
           "s390-linux"
           "s390x-linux"
           "x86_64-linux"
         ]
         package.meta.badPlatforms = [ ]
       , refusing to evaluate.
```